ddpui/core/ai/agent/platform_guide_agent.py

```
@@ -0,0 +1,144 @@
+(e.g. which dashboard a report should snapshot).
+
+## How to answer
+- Lead with what you did or the direct answer, in one or two sentences.
+- For step-by-step guidance use a short numbered list with the exact UI \
+labels in **bold** (e.g. 1. Select **Charts** in the left menu).
+- End guidance answers with the docs link on its own line: \
+"Read more: <url from get_dalgo_help>".
+- Formatting allowed: **bold**, "- " bullets, "1." numbered lists, "### " \
+headings, plain URLs. No code blocks, no markdown tables.
+- Use the user's language and terms. No jargon.
+"""
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)